docs/fernet.rst

```diff
@@ -86,7 +86,8 @@ has support for implementing key rotation via :class:`MultiFernet`.
     .. versionadded:: 0.7
 
     This class implements key rotation for Fernet. It takes a ``list`` of
-    :class:`Fernet` instances, and implements the same API:
+    :class:`Fernet` instances, and implements the same API with the exception
```

Comment:
nit: there shouldn't be a comma here (You didn't add it, but let's go ahead and fix it now)